oms-kemp-applicationdelivery-solution/azuredeploy.json

```diff
@@ -0,0 +1,906 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsLogAnalyticsWorkspaceName": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "Create new or refer to an existing OMS Log Analytics Workspace"
+            }
+        },
+        "omsLogAnalyticsRegion": {
+            "type": "string",
+            "defaultValue": "eastus",
+            "allowedValues": [
+                "westeurope",
+                "eastus",
+                "southeastasia"
+            ],
+            "metadata": {
+                "description": "Specify the Azure Region for your new or existing OMS workspace"
+            }
+        },
+        "omsLogAnalyticsSku": {
+            "type": "string",
+            "defaultValue": "free",
+            "allowedValues": [
+                "free",
+                "standalone",
+                "pernode"
+            ],
+            "metadata": {
+                "description": "Specify the Azure Region for your OMS Automation Account"
+            }
+        }        
+    },
+    "variables": {
+        "omsSolutions": {
+            "customSolution": {
+                "viewName": "KempApplicationDelivery",
+                "solutionName": "[concat('KempApplicationDelivery', '[', parameters('omsLogAnalyticsWorkspaceName'), ']')]",
+                "publisher": "QND",
+                "displayName": "Kemp Application Delivery",
+                "description": "Monitor and analyze your Kemp Application Delivery devices",
+                "author": "daniele.grandini@live.it",
+                "version": "1",
+                "product": "OMSGallery/KempApplicationDelivery"
+            }
+        },        
+        "omsLogAnalyticsAPIVersion": "2015-11-01-preview", 
+        "querytemplateuri": "https//rawcontent.github.com",
+        "location": "[parameters('omsLogAnalyticsRegion')]", 
```

Comment:
why is this put into a variable, instead of referencing the parameter in the template?